snuba/query/processors/granularity_processor.py

```diff
@@ -48,11 +48,11 @@ class GranularityMapping(NamedTuple):
     enum_value: int
 
 
-PERFORMANCE_GRANULARITIES: Sequence[GranularityMapping] = [
-    GranularityMapping(60, 1),
-    GranularityMapping(3600, 2),
-    GranularityMapping(86400, 3),
-]
+PERFORMANCE_GRANULARITIES: Mapping[int, int] = {
+    60: 1,
+    3600: 2,
+    86400: 3,
+}
```

Comment:
this `int, int` mapping makes the object hierarchy simpler but as a reader of the code, it's not clear to me what `PERFORMANCE_GRANULARITIES` are

Reply:
That's fair but when the move to YAML is complete this constant will be deleted.

Reply:
> Would a doc comment on the constructor help?

Yeah that would be good, just so as the reader I can figure out what that dict is for. The constructor is not going away either way

Reply:
added